DZSwiftUI/DZSwiftUI/ViewModel/PaymentViewModel.swift

```diff
@@ -0,0 +1,131 @@
+// PaymentViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Вью модель экрана описания товара
+final class PaymentViewModel: ObservableObject {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let emptyString = ""
+        static let startIndex = 0
+        static let maximumCVCChars = 3
+        static let maximumCardNumberChars = 16
+        static let startBackFlipDegree = -90.0
+        static let startFrontFlipDegree = 0.0
+    }
+
+    // MARK: - Public Properties
+
+    @Published var isAlertButtonShown = false
+    @Published var isFlipCardShown = false
+    @Published var isCardNumberAnimationShown = false
+    @Published var isCardholderAnimationShown = false
+    @Published var isCardCVCAnimationShown = false
+    @Published var isMonthPickerAnimationShown = false
+    @Published var isYearPickerAnimationShown = false
+    @Published var backFlipDegree = Constants.startBackFlipDegree
+    @Published var frontFlipDegree = Constants.startFrontFlipDegree
+    @Published var isAlertCVCShown = false
+    @Published var cardNumberText = Constants.emptyString
+    @Published var cardholderText = Constants.emptyString
+    @Published var cardCVCText = Constants.emptyString
+    @Published var monthPickerIndex = Constants.startIndex
+    @Published var yearPickerIndex = Constants.startIndex
+    @Published var lastCharCVCText = Constants.emptyString
+    @Published var lastCharCardNumberText = Constants.emptyString
+    @Published var monthPickerTexts = [
+        "00",
+        "01",
+        "02",
+        "03",
+        "04",
+        "05",
+        "06",
+        "07",
+        "08",
+        "09",
+        "10",
+        "11",
+        "12"
+    ]
+    @Published var yearPickerTexts = [
+        "00",
+        "23",
+        "24",
+        "25",
+        "26",
+        "27",
+        "28"
+    ]
```

Comment:
```suggestion
    @Published var yearPickerTexts = ["00", "23", "24", "25", "26", "27", "28"
    ]
```
 и так нормально читается